### ISSUE TITLE
How can I stream in python?

### ISSUE THREAD

**Issue by martamill3** [joshvillbrandt/goprohero#21]
Hi,
This library is awesome! I would like to stream the recording images in my python interface.. is this possible?

Thank you so much

**Comment by arpit1997** [comment]
@martamill3 Yeah it is possible but you have to write some additional codes for this here are some hints.
There are many libraries available you can use [gst-python](https://gstreamer.freedesktop.org/modules/gst-python.html) also.
There is another fmous library named [python-requests](http://github.com/kennethreitz/requests)
A sample code which you can use:

```
>>> import requests
>>> url = <any url >
>>> r = requests.get(url, stream=True)
```

then pass this r object to any interface it will stream the video.
Hope this helps :smile: :smile_cat:

**Comment by ghost** [comment]
Hi! I really don't get the point of how to stream the images in TKinter. I've OpenCV installed but when I pass the url (http://10.5.5.9:8080/gp/gpControl/execute?p1=gpStream&c1=restart) and the R object nothing appears and I don't get any error so I'm quite lost..

Thanks so much
